docs/index.rst

```diff
@@ -8,6 +8,8 @@ Contents:
 .. toctree::
    :maxdepth: 3
 
-   jdbc_connector
-   configuration_options
+   source_connector
```

Comment:
Hmm, no strong preference from me but the current number of pages doesn't warrant adding an additional level of hierarchy I think. I'll change the ordering though so it's [source_connector, source_config_options, sink_connector, sink_config_options